.github/renovate/commitMessage.json5

```diff
@@ -0,0 +1,18 @@
+{
+  "commitMessagePrefix": ":seedling: ",
+  "commitMessageTopic": "{{depName}}",
+  "commitMessageExtra": "to {{newVersion}}",
+  "commitMessageSuffix": "",
+  "group": { commitMessageTopic: "{{{groupName}}} group" },
+  "packageRules": [
+    {
+      "matchDatasources": ["helm"],
+      "commitMessageTopic": "chart {{depName}}"
+    },
+    {
+      "matchDatasources": ["docker"],
+      "commitMessageTopic": "image {{depName}}",
+      "commitMessageExtra": "to {{#if isSingleVersion}}v{{{newVersion}}}{{else}}{{{newValue}}}{{/if}}"
+    }
+  ]
+}
```

Comment:
The new line at the end file is missing. Please also check other files.

Reply:
@michal-gubricky I've question around why do we enforce this in SCS repo? We have one lint target which gives us an error. 

```bash
$ make lint-yaml
docker run  --rm -t -i \
	-v /home/k7/go/pkg:/go/pkg \
	-v /home/k7/work/scsoss/cluster-stack-operator:/src/cluster-stack-operator \
	ghcr.io/sovereigncloudstack/cso-builder:1.0.0 lint-yaml;
make: Entering directory '/src/cluster-stack-operator'
yamllint --version
yamllint 1.30.0
yamllint -c .yamllint.yaml --strict .
./.github/workflows/schedule-update-bot.yaml
  58:1      error    too many blank lines (1 > 0)  (empty-lines)
```

I'm not against it but just wanted to ask if there's a reason behind this.

Reply:
https://github.com/SovereignCloudStack/cluster-stack-provider-openstack/pull/93#discussion_r1505463638

Reply:
@michal-gubricky updated it via this commit https://github.com/SovereignCloudStack/cluster-stack-operator/commit/d9fe0f55c979a8bcc8801692beed86524b320c5d 

I think GitHub is only highlighting it for YAML files not for files with `json5` extension. I've that extra line in my editor locally. 

Please check the attached screenshot: 

<details><summary>Details</summary>
<p>

![image](https://github.com/SovereignCloudStack/cluster-stack-operator/assets/81210977/53e82a6c-8d48-40c8-a904-9f0f91471a10)


</p>
</details>

Reply:
> I think GitHub is only highlighting it for YAML files not for files with `json5` extension. I've that extra line in my editor locally.

I think GitHub highlights all files when there is a blank line missing at the end of the file. That is why I raised this issue, because I saw it in `files changed` tab.

> updated it via this commit https://github.com/SovereignCloudStack/cluster-stack-operator/commit/d9fe0f55c979a8bcc8801692beed86524b320c5d

Everything looks good from my side now. Thank you @kranurag7.